### PR TITLE
[github-actions] add OTNS stress tests

### PR DESCRIPTION
**Pull request by simonlingoogle**
This commit runs OTNS stress tests in OpenThread CI.

### CODE REVIEW

**Review by wgtdkp — APPROVED** [review]
LGTM 👍

File: .github/workflows/otns.yml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  cancel-previous-runs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: rokroskar/workflow-run-cleanup-action@master
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      if: "github.ref != 'refs/heads/master'"
+
+  unittests:
+    runs-on: ubuntu-18.04
```
Comment (jwhui) [inline .github/workflows/otns.yml:52]: Update to 20.04?

File: .github/workflows/otns.yml [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+name: OTNS
+
+on: [push, pull_request]
+
+env:
+  OTNS_COMMIT: "5a3d6c953eb45d9ed5704b4e89a572853ea94bbc" # Oct 26, 2020
```
Comment (jwhui) [inline .github/workflows/otns.yml:34]: Update to be consistent with #5899 ?

**Comment by simonlingoogle** [comment]
@jwhui All CI passed. One job is removed, so will never finish.

**Review by jwhui — APPROVED** [review]

**Pull request merged**